Libs/vtkITK/vtkITKImageWriter.cxx

```diff
@@ -64,15 +64,15 @@ void ITKWriteVTKImage(vtkITKImageWriter *self, vtkImageData *inputImage, char *f
   typename ImageType::PointType origin;
   direction.SetIdentity();
 
-  double mag[3];
-  int i;
+  double mag[3] = {0.0};
+  int i = 0;
   for (i=0; i<3; i++)
```

Comment:
It would be nicer to have `int i=0` in the `for` loop. Very small thing, not required to be fixed in this PR, just if you have time.